### PR TITLE
cwltool bump and temporary GPU test disablement [BA-6535]

### PR DESCRIPTION
**Pull request by mcovarr**
~Draft PR that may not be 100% working. The sub-build short circuiting appears to be not quite right and prevents the conformance tests from running even with [force ci].~

Ready for review, now with frankenbranch GPU test disablement.

### CODE REVIEW

File: src/ci/bin/test.inc.sh [hunk 1]
```diff
@@ -626,7 +626,7 @@ cromwell::private::create_centaur_variables() {
 
 cromwell::private::create_conformance_variables() {
     CROMWELL_BUILD_CWL_RUNNER_MODE="${CROMWELL_BUILD_BACKEND_TYPE}"
-    CROMWELL_BUILD_CWL_TOOL_VERSION="1.0.20190228155703"
+    CROMWELL_BUILD_CWL_TOOL_VERSION="1.0.20191225192155"
```
Comment (cjllanwarne) [inline src/ci/bin/test.inc.sh:629]: Still `2019...`?

Comment (cjllanwarne) [reply src/ci/bin/test.inc.sh:629]: Although... `20191225...` - Merry Christmas CWLtool!

Comment (mcovarr) [reply src/ci/bin/test.inc.sh:629]: Yeah, I suspect we're supposed to be upgrading to a Python 3 version as Python 2 officially deprecated on Jan 1 2020?


ERROR: Could not find a version that satisfies the requirement cwltool==3.0.20200724003302 (from versions: 0.1.20141118163626.cbb1b06, 0.1.20141118185349.d594c3e, 0.1.20141119031726.8dcd753, 0.1.20141125184103.2d72b5c, 0.1.20141128210437.24bfcbd, 0.1.20141205194642.2cec11a, 0.1.20150123125045.772314e, 0.1.20150123213120.f1cfee1, 1.0, 1.0.20150317015350, 1.0.20150318010828, 1.0.20150318015654, 1.0.20150324011020, 1.0.20150324012706, 1.0.20150324172633, 1.0.20150324173916, 1.0.20150324194338, 1.0.20150324200038, 1.0.20150324203244, 1.0.20150324211351, 1.0.20150326005309, 1.0.20150326211508, 1.0.20150327211418, 1.0.20150331014533, 1.0.20150401014645, 1.0.20150403144650, 1.0.20150407211342, 1.0.20150409193818, 1.0.20150415152714, 1.0.20150429022405, 1.0.20150508023316, 1.0.20150512154247, 1.0.20150512164512, 1.0.20150522025529, 1.0.20150525010411, 1.0.20150527172438, 1.0.20150601172915, 1.0.20150601200103, 1.0.20150602212140, 1.0.20150605134212, 1.0.20150606210100, 1.0.20150609024148, 1.0.20150609205603, 1.0.20150610150530, 1.0.20150621213723, 1.0.20150621230545, 1.0.20150622134207, 1.0.20150622144406, 1.0.20150622183905, 1.0.20150623020304, 1.0.20150623031442, 1.0.20150624181807, 1.0.20150624201710, 1.0.20150626032845, 1.0.20150626035015, 1.0.20150626035417, 1.0.20150627015636, 1.0.20150706174116, 1.0.20150707043955, 1.0.20150708115452, 1.0.20150709135652, 1.0.20150711153503, 1.0.20150713135958, 1.0.20150713152138, 1.0.20150713185212, 1.0.20150714132635, 1.0.20150715173345, 1.0.20150720194125, 1.0.20150722144138, 1.0.20150723141036, 1.0.20150728161219, 1.0.20150808190318, 1.0.20150813132038, 1.0.20150814015745, 1.0.20150814191928, 1.0.20150910021332, 1.0.20150915034626, 1.0.20150916041152, 1.0.20150916124227, 1.0.20150918080732, 1.0.20150923183439, 1.0.20150925123259, 1.0.20150929170517, 1.0.20151007133346, 1.0.20151009020717, 1.0.20151012161407, 1.0.20151012173709, 1.0.20151013134821, 1.0.20151013135545, 1.0.20151013173827, 1.0.20151022085449, 1.0.20151026181844, 1.0.20151104062100, 1.0.20151110030107, 1.0.20151112194920, 1.0.20151120220905, 1.0.20151121025646, 1.0.20151121032923, 1.0.20151122025918, 1.0.20151124040259, 1.0.20151124220039, 1.0.20151125172224, 1.0.20151125211848, 1.0.20151125221324, 1.0.20151130190253, 1.0.20151130204648, 1.0.20151209160516, 1.0.20151210154014, 1.0.20151211141743, 1.0.20151211155311, 1.0.20160108161501, 1.0.20160108200940, 1.0.20160115132942, 1.0.20160126152227, 1.0.20160126211726, 1.0.20160127144612, 1.0.20160128142049, 1.0.20160129152024, 1.0.20160129161544, 1.0.20160129183049, 1.0.20160203144931, 1.0.20160203221531, 1.0.20160209222805, 1.0.20160222205901, 1.0.20160225033850, 1.0.20160225040942, 1.0.20160225202307, 1.0.20160226205952, 1.0.20160302134341, 1.0.20160304012934, 1.0.20160307200623, 1.0.20160308152645, 1.0.20160309204538, 1.0.20160310140736, 1.0.20160310215251, 1.0.20160311170456, 1.0.20160311201238, 1.0.20160315233236, 1.0.20160316150250, 1.0.20160316204054, 1.0.20160322201127, 1.0.20160323212343, 1.0.20160325200114, 1.0.20160325210917, 1.0.20160331184641, 1.0.20160408133555, 1.0.20160411021840, 1.0.20160411194133, 1.0.20160412203002, 1.0.20160413143011, 1.0.20160415153333, 1.0.20160421140153, 1.0.20160421171618, 1.0.20160422203349, 1.0.20160422204730, 1.0.20160425140546, 1.0.20160426185417, 1.0.20160427142240, 1.0.20160504183010, 1.0.20160505211047, 1.0.20160507101510, 1.0.20160510161706, 1.0.20160511142738, 1.0.20160511162129, 1.0.20160511213450, 1.0.20160515155531, 1.0.20160518200809, 1.0.20160518201549, 1.0.20160519182434, 1.0.20160523144113, 1.0.20160531173804, 1.0.20160608124624, 1.0.20160609160402, 1.0.20160610111115, 1.0.20160614212644, 1.0.20160616182520, 1.0.20160623183600, 1.0.20160626203316, 1.0.20160627152300, 1.0.20160628194545, 1.0.20160629140624, 1.0.20160630171631, 1.0.20160706132520, 1.0.20160707195347, 1.0.20160708190014, 1.0.20160712154127, 1.0.20160714182449, 1.0.20160726135535, 1.0.20160805221855, 1.0.20160810161358, 1.0.20160810200423, 1.0.20160811184335, 1.0.20160829192223, 1.0.20160829211335, 1.0.20160901133827, 1.0.20160907141844, 1.0.20160912182208, 1.0.20160913171024, 1.0.20160915202519, 1.0.20160918012352, 1.0.20160919152321, 1.0.20160922135240, 1.0.20160923180109, 1.0.20160930152149, 1.0.20161005195021, 1.0.20161007181528, 1.0.20161107145355, 1.0.20161114152756, 1.0.20161115095848, 1.0.20161122201220, 1.0.20161123190203, 1.0.20161128202906, 1.0.20161202203310, 1.0.20161206195941, 1.0.20161206204434, 1.0.20161207161158, 1.0.20161216212910, 1.0.20161221171240, 1.0.20161223144155, 1.0.20161227200419, 1.0.20170105144051, 1.0.20170111193653, 1.0.20170112154257, 1.0.20170112185927, 1.0.20170114120503, 1.0.20170118141124, 1.0.20170119182607, 1.0.20170119234115, 1.0.20170213175853, 1.0.20170217172322, 1.0.20170224141733, 1.0.20170308174714, 1.0.20170309164828, 1.0.20170327143622, 1.0.20170329142446, 1.0.20170413151007, 1.0.20170413194156, 1.0.20170510151339, 1.0.20170510165748, 1.0.20170516234254, 1.0.20170525215327, 1.0.20170622090721, 1.0.20170629171139, 1.0.20170704143016, 1.0.20170707200431, 1.0.20170713144155, 1.0.20170713151519, 1.0.20170717120410, 1.0.20170717200612, 1.0.20170718140316, 1.0.20170721130823, 1.0.20170721160741, 1.0.20170721221557, 1.0.20170723124118, 1.0.20170727112954, 1.0.20170803160545, 1.0.20170810192106, 1.0.20170811195303, 1.0.20170815202200, 1.0.20170817131858, 1.0.20170822192924, 1.0.20170828135420, 1.0.20170927182241, 1.0.20170928192020, 1.0.20171017195544, 1.0.20171107133715, 1.0.20171221100033, 1.0.20171227212058, 1.0.20180108222053, 1.0.20180111185617, 1.0.20180116032016, 1.0.20180116213856, 1.0.20180130110340, 1.0.20180211183944, 1.0.20180220121300, 1.0.20180224111654, 1.0.20180224135035, 1.0.20180225105849, 1.0.20180302231433, 1.0.20180306140409, 1.0.20180306163216, 1.0.20180322194411, 1.0.20180326152342, 1.0.20180330141240, 1.0.20180403145700, 1.0.20180502225535, 1.0.20180508202931, 1.0.20180518123035, 1.0.20180521150620, 1.0.20180522135731, 1.0.20180524215209, 1.0.20180525185854, 1.0.20180615183820, 1.0.20180618135130, 1.0.20180622214234, 1.0.20180711112827, 1.0.20180721142728, 1.0.20180806154020, 1.0.20180806194258, 1.0.20180808165120, 1.0.20180809224403, 1.0.20180819175200, 1.0.20180820141117, 1.0.20180906112546, 1.0.20180912090223, 1.0.20180921163344, 1.0.20180923172926, 1.0.20181012180214, 1.0.20181102182747, 1.0.20181114114354, 1.0.20181115163053, 1.0.20181115172406, 1.0.20181116032456, 1.0.20181118133959, 1.0.20181129095638, 1.0.20181201184214, 1.0.20181217162649, 1.0.20190228155703, 1.0.20190525125941, 1.0.20190528194036, 1.0.20190603140227, 1.0.20190607183319, 1.0.20190618201008, 1.0.20190621234233, 1.0.20190813131033, 1.0.20190815141648, 1.0.20190831161204, 1.0.20190906054215, 1.0.20190915164430, 1.0.20191022103248, 1.0.20191206125148, 1.0.20191225192155)

Comment (mcovarr) [reply src/ci/bin/test.inc.sh:629]: Yup that seems to be it, I can `pip3 install` the cwltool version that begat that error message but I can't `pip2 install` it.

**Comment by mcovarr** [comment]
tracking ticket for GPU issues: https://broadworkbench.atlassian.net/browse/BA-6540

**Pull request merged**